### PR TITLE
Add Account portal

### PR DESCRIPTION
**Pull request by aospiridonov**
Add Account portal

### CODE REVIEW

File: example/account.dart [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import 'package:xdg_desktop_portal/xdg_desktop_portal.dart';
+
+void main(List<String> args) async {
+  var client = XdgDesktopPortalClient();
+  //var userInformation = await client.account.getUserInformation().first;
```
Comment (robert-ancell) [inline example/account.dart:5]: Please drop unused code.

File: lib/src/xdg_desktop_portal_client.dart [hunk 2]
```diff
@@ -95,6 +95,65 @@ abstract class _XdgPortalSession extends DBusRemoteObject {
   Future<void> _handleClosed();
 }
 
+class XdgAccountUserInformation {
+  ///The user id.
```
Comment (robert-ancell) [inline lib/src/xdg_desktop_portal_client.dart:99]: Missing space, i.e. `/// The`

File: lib/src/xdg_desktop_portal_client.dart [hunk 3]
```diff
@@ -95,6 +95,65 @@ abstract class _XdgPortalSession extends DBusRemoteObject {
...
+  String id;
+
+  ///The users real name.
+  String name;
+
+  ///The uri of an image file for the users avatar photo.
```
Comment (robert-ancell) [inline lib/src/xdg_desktop_portal_client.dart:105]: uri -> URI

File: lib/src/xdg_desktop_portal_client.dart [hunk 4]
```diff
@@ -95,6 +95,65 @@ abstract class _XdgPortalSession extends DBusRemoteObject {
   Future<void> _handleClosed();
 }
 
+class XdgAccountUserInformation {
+  ///The user id.
+  String id;
```
Comment (robert-ancell) [inline lib/src/xdg_desktop_portal_client.dart:100]: These can all be final.

File: lib/src/xdg_desktop_portal_client.dart [hunk 5]
```diff
@@ -1120,6 +1179,9 @@ class XdgDesktopPortalClient {
   final _requests = <DBusObjectPath, _XdgPortalRequest>{};
   final _sessions = <DBusObjectPath, _XdgPortalSession>{};
 
+  /// Portal for obtaining information about the user
```
Comment (robert-ancell) [inline lib/src/xdg_desktop_portal_client.dart:1182]: `.` on the end of the comment.

File: test/xdg_desktop_portal_test.dart [hunk 6]
```diff
@@ -459,6 +517,10 @@ class MockPortalServer extends DBusClient {
   final sessions = <MockPortalSessionObject>[];
   final usedTokens = <String>{};
 
+  final String? id;
```
Comment (robert-ancell) [inline test/xdg_desktop_portal_test.dart:520]: Recommend naming these `userId`, `userName` etc so it's clear what these are used for.

**Comment by robert-ancell** [comment]
Thanks for this! Can you rebase so this is a single commit? Also please add `Fixes https://github.com/canonical/xdg_desktop_portal.dart/issues/1` to the commit message.

**Comment by aospiridonov** [comment]
> 
Please help me! How to rebase so this is a single commit?

**Comment by aospiridonov** [comment]
> Thanks for this! Can you rebase so this is a single commit? Also please add `Fixes https://github.com/canonical/xdg_desktop_portal.dart/issues/1` to the commit message.

Please help me! How to rebase so this is a single commit?

**Comment by aospiridonov** [comment]
fix comments PR

**Comment by aospiridonov** [comment]
> > Thanks for this! Can you rebase so this is a single commit? Also please add `Fixes https://github.com/canonical/xdg_desktop_portal.dart/issues/1` to the commit message.
> 
> Please help me! How to rebase so this is a single commit?
> 
> From the command line: `git rebase main` or `git rebase -i main` to merge together any patches.
after git rebase main
write "HEAD detached at de3405e"

**Comment by robert-ancell** [comment]
> > > Thanks for this! Can you rebase so this is a single commit? Also please add `Fixes https://github.com/canonical/xdg_desktop_portal.dart/issues/1` to the commit message.
> > 
> > 
> > Please help me! How to rebase so this is a single commit?
> > From the command line: `git rebase main` or `git rebase -i main` to merge together any patches.
> 
> after git rebase main write "HEAD detached at [de3405e](https://github.com/canonical/xdg_desktop_portal.dart/commit/de3405e19a8d76c1e0145353531fcfc4275b6619)" What do I need to do next?

Once you have all the changes as you expect, then use `git add` to accept those changes and then `git rebase --continue` to finish the rebase.

**Comment by aospiridonov** [comment]
> > > > Thanks for this! Can you rebase so this is a single commit? Also please add `Fixes https://github.com/canonical/xdg_desktop_portal.dart/issues/1` to the commit message.
> > > 
> > > 
> > > Please help me! How to rebase so this is a single commit?
> > > From the command line: `git rebase main` or `git rebase -i main` to merge together any patches.
> > 
> > 
> > after git rebase main write "HEAD detached at [de3405e](https://github.com/canonical/xdg_desktop_portal.dart/commit/de3405e19a8d76c1e0145353531fcfc4275b6619)" What do I need to do next?
> 
> Once you have all the changes as you expect, then use `git add` to accept those changes and then `git rebase --continue` to finish the rebase.

I can't do it. Could you help me? I can add you as a contributor to my repository.

**Comment by robert-ancell** [comment]
> I can't do it. Could you help me? I can add you as a collaborators to my repository.

All done, should be good to land now :)

**Comment by aospiridonov** [comment]
> > I can't do it. Could you help me? I can add you as a collaborators to my repository.
> 
> All done, should be good to land now :)

You are wonderful! Thank you very much)

**Pull request merged**